src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -481,25 +502,28 @@
 short-summary: List the tasks for an Azure Container Registry.
 examples:
   - name: List tasks and show the results in a table.
-    text: >
+    text: >-
         az acr task list -r MyRegistry -o table
+  - name: List all of the executed runs for an Azure Container Registry, with the ability to filter by a specific Task
+    text: az acr task list-runs  --registry MyRegistry
```

Comment:
This example doesn't even use the `az acr task list` command. This should throw a linter error.

Reply:
However, if it was in the correct locations, I would not say that this example is better than any of the examples already in place for that command.